### DISCUSSION TITLE
On-prem bundle fragment not deploying

### DISCUSSION THREAD

**Discussion by jdavenport-vervio** [Adobe-Consulting-Services/acs-aem-commons#3496] [Q&A] [answered]
Hello all,

I'm working on an upgrade to service pack 22 and the latest ACS Commons version. One of our integrations which uses the EmailService is not working because the reference to `com.adobe.acs.commons.email.impl.MailTemplateManager` is unsatisfied. After looking in the code I noticed this class is implemented in two respective bundle fragments, `bundle-onprem` and `bundle-cloud`.

<img width="772" alt="Screenshot 2024-12-17 at 5 24 34 pm" src="https://github.com/user-attachments/assets/6bf9d84c-cbd3-4aa3-83fb-a083ddc98522" />

Since we are on on-prem I would expect the on-prem fragment to be deployed, and it appears this is meant to be deployed by the all package for acs commons. However it is not being deployed on my environment and the jar is not under `/apps/acs-commons/install` in CRXDE.

Are there any additional steps I need to take to ensure this bundle fragment is deployed correctly?

Thanks

**Comment by kwin** [comment, marked as answer]
This is a known issue (https://github.com/Adobe-Consulting-Services/acs-aem-commons/issues/3471) and will be fixed in the next version.

**Reply by jdavenport-vervio** [reply]
Thanks @kwin, when will the fix be released?

**Reply by kwin** [reply]
Currently only @davidjgonzalez has permissions to release so only he can answer.
